src/TTGO_T-Beam_LoRa_APRS.ino

```diff
@@ -882,6 +893,17 @@ void loop() {
     }
   }
 
+  // Show informations on WiFi Status
+  if (apConnected) {
+    enableOled(); // turn ON OLED temporary
+    writedisplaytext(" ((WiFi))","WiFi Client Mode","SSID: " + infoApName, "Pass: ********", "IP: " + infoApAddr, getSatAndBatInfo());
+    apConnected=false;
+  } else if (apEnabled) {
+    enableOled(); // turn ON OLED temporary
+    writedisplaytext(" ((WiFi))","WiFi AP Mode","SSID: " + infoApName, "Pass: " + infoApPass, "IP: " + infoApAddr, getSatAndBatInfo());
+    apEnabled=false;
+  }
```

Comment:
Showing WiFi informations for few seconds

Reply:
Thank you for this!